backup_and_restore/backing-up-etcd.adoc

```diff
@@ -15,6 +15,8 @@ installation, otherwise the backup will contain expired certificates. It is also
 recommended to take etcd backups during non-peak usage hours, as it is a
 blocking action.
 
+It is important to take an etcd backup after you upgrade your cluster, because when restoring, you must use an etcd backup from the same z-stream release as the current cluster. For example, an {product-title} 4.5.2 cluster must use an etcd backup that was taken from 4.5.2.
```

Comment:
@hexfusion I figure we should give a heads up on the etcd backup page as well. Is this okay to add (suggesting to take a backup after an upgrade)?

Reply:
Regular backup should be recommended.

Reply:
@hexfusion so are you saying that we don't need this addition here?